frontend/mms-Admin/components/componentStyles/profileimage.module.css

```diff
@@ -0,0 +1,175 @@
+.profileImage_main {
+  width: 18rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.profileImage_card {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  overflow: hidden;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.2);
+  margin-top: 1rem;
+}
+
+.profileImage_card img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.profileImage_sub_upload p {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 33px;
+  color: #333;
+}
+
+.profileImage_upload_button {
+  padding: 10px 20px;
+  background-color: #058b94;
+  color: #fff;
+  font-size: 16px;
+  font-weight: bold;
+  border-radius: 5px;
+  cursor: pointer;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.2);
+}
+
+.profileImage_upload_button input[type="file"] {
+  display: none;
+}
+
+.Name_div {
+  padding-top: 3rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.lastName,
+.firstName {
+  width: 425.78px;
+  height: 50px;
+  font-size: 1.2rem;
+  padding: 5px;
+  margin-bottom: 20px;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.2);
+}
+
+.fullName_div {
+  padding-top: 0.5rem;
+}
+
+.social_name_div label,
+.web_sub_div label,
+.cityname_div label,
+.countryname_div label,
+.div_about label,
+.fullName_div label {
+  font-size: 1.2rem;
+}
+
+.bio {
+  width: 915px;
```

Comment:
this assumes, that your design is not responsive

Reply:
it is